daemon/container_config.c

```diff
@@ -572,7 +572,8 @@ container_config_get_vnet_cfg_list_new(const container_config_t *config)
 	}
 
 	if (if_cfg_list == NULL) {
-		list_t *nw_name_list = hardware_get_nw_name_list();
+		list_t *nw_name_list = NULL;
+		nw_name_list = list_append(nw_name_list, "eth0");
```

Comment:
here we can rename the interface to "host0" this is currently common practice in systemd environments
and maybe we introduce a define for this, e.g., CONTAINER_CONFIG_DEFAULT_VETH_NAME

Reply:
done